release-notes/unreleased/2494.update.rst

```diff
@@ -0,0 +1,2 @@
+Add a new ``merge_common_instructions`` input argument to :meth:`draw_circuit_schedule_timing` that 
+if set to ``True`` will merge instructions of the same type based on temporal continuity.
```

Comment:
I think that adding the example of how the merge works from the PR description would be really helpful for the release note, this would allow users to reference it in an easier way in the future.